readme.txt

```diff
@@ -13,17 +13,17 @@ Ecommerce Analytics and behaviour-driven customer engagement tools for ecommerce
 
 [Metrilo](https://www.metrilo.com/?ref=wpplugin) is a growth platform for the modern product brands with a long-term strategy and looking for profitability. Metrilo offers tools for marketing optimization, customer management, engagement and retention as well as email and feedback gathering automation. It facilitates growth through usable data and unique focus on driving repeat sales. Tailored to be used by brand founders, agencies and small marketing teams.
 
-The extension connects easily to your Woocommerce store and processes sales and marketing data into ready-to-use reports and rich customer insights, including historical data - Metrilo will make past orders and old customers accessible for reactivation, too! Then, you can monitor how your store is doing via the dashboard with all key metrics in real time. The Marketing performance, Revenue breakdown and Product performance reports show opportunities for optimization. The Customer Database makes your first-party customer data usable for segmentation and engagement via emails or retargeting. The Customer Retention feature helps you understand the customer lifecycle and loyalty to drive more repeat sales. And finally, the emails - manual and automated, utilize all data to reach customers at the right time with the right message. Metrilo opens up opportunities to grow your revenue without spending more on marketing.
+The extension connects easily to your WooCommerce store and processes sales and marketing data into ready-to-use reports and rich customer insights, including historical data - Metrilo will make past orders and old customers accessible for reactivation, too! Then, you can monitor how your store is doing via the dashboard with all key metrics in real time. The Marketing performance, Revenue breakdown and Product performance reports show opportunities for optimization. The Customer Database makes your first-party customer data usable for segmentation and engagement via emails or retargeting. The Customer Retention feature helps you understand the customer lifecycle and loyalty to drive more repeat sales. And finally, the emails - manual and automated, utilize all data to reach customers at the right time with the right message. Metrilo opens up opportunities to grow your revenue without spending more on marketing.
```

Comment:
`The Marketing performance, Revenue and Products reports show opportunities for optimization.`

`The Retention feature helps you understand the customer lifecycle and loyalty to drive more repeat sales.`